src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts

```diff
@@ -116,13 +112,18 @@ export class DataMapperHostComponent extends FlowPage implements OnInit, OnDestr
     if (mappings) {
       const mappingDefinition = new MappingDefinition();
       // Existing mappings, load from the route
+      try {
       this.mappingService.deserializeMappingServiceJSON(mappings, mappingDefinition);
+      } catch (err) {
+        // TODO popup or error alert?  At least catch this so we initialize
```

Comment:
Definitely, we'll need that global notification system in place or at least an inline notification.